api/src/test/resources/org/openmrs/api/include/ConditionServiceImplTest-SetupCondition.xml

```diff
@@ -11,17 +11,21 @@
 
 -->
 <dataset>
-	<conditions condition_id="1" patient_id="2" clinical_status="INACTIVE" creator="1" date_created="2015-01-12 00:00:00" 
+
+	<encounter encounter_id="2039" encounter_type="2" patient_id="2" location_id="1" form_id="1" encounter_datetime="2020-04-20 14:09:05.0" 
+				creator="1" date_created="2020-04-20 14:09:05.0" voided="false" uuid="0519d652-393b-4118-9c83-a3715b82d4av"/>
+
+	<conditions condition_id="1" patient_id="2" clinical_status="INACTIVE" encounter_id="2039" creator="1" date_created="2015-01-12 00:00:00" 
 			   voided="false" onset_date="2017-01-12 00:00:00" end_date="2017-01-15 00:00:00" verification_status="PROVISIONAL" 
 			   uuid="2cc6880e-2c46-15e4-9038-a6c5e4d22fb7"/>
 	<conditions condition_id="2" patient_id="2" clinical_status="ACTIVE" creator="1" date_created="2015-01-12 00:00:00" 
 			   voided="false" onset_date="2017-01-12 00:00:50" verification_status="CONFIRMED" 
 			   uuid="2cc6880e-2c46-11e4-9138-a6c5e4d20fb7"/>
-	<conditions condition_id="4" patient_id="2" clinical_status="ACTIVE" creator="1" date_created="2016-01-12 00:00:00" 
```

Comment:
I get scared when you add an encounter_id to existing conditions. It looks as if they no longer save without encounters, yet they are optional.

Reply:
Resolved @dkayiwa